### DISCUSSION TITLE
Kons-9/Scheme

### DISCUSSION THREAD

**Discussion by ghost** [shirok/Gauche#839] [General]
[edit]

Probably just going to use my own library

**Comment by shirok** [comment]
Ho! It's a joyful surprise.  I know Kaveh for long time; We have worked on number of CG/videogame projects .  I've been eyeing on his Kons-9 since its inception.  Glad to know it gets mroe traction.  His code is heavily depend on CLOS, so targeting at Scheme with CLOS-like OO would be interesting.
